control-plane-operator/controllers/hostedcontrolplane/hostedcontrolplane_controller.go

```diff
@@ -1218,6 +1221,225 @@ func (r *HostedControlPlaneReconciler) reconcile(ctx context.Context, hostedCont
 		return fmt.Errorf("failed to reconcile default security group")
 	}
 
+	// TODO: consider using a side container in the etcd pod instead.
+	r.Log.Info("Reconciling etcd-backup cronJob")
+	if hostedControlPlane.Spec.Platform.Type == hyperv1.AWSPlatform {
```

Comment:
Should the creation of the CronJob be conditioned on whether the configmap `etcd-backup-config` exists?